_inc/client/security/jetpack-backup.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { LoadingCard } from './loading-card';
+import Banner from 'components/banner';
+import SettingsCard from 'components/settings-card';
+
+export class JetpackBackup extends Component {
+	static propTypes = {
+		siteRawUrl: PropTypes.string,
+		rewindState: PropTypes.string,
+	};
+
+	static defaultProps = {
+		siteRawUrl: '',
+		rewindState: '',
+	};
+
+	getRewindMessage() {
+		const { siteRawUrl, rewindStatus } = this.props;
+		const rewindState = get( rewindStatus, [ 'state' ], false );
+
+		switch ( rewindState ) {
+			case 'provisioning':
+				return {
+					title: __( 'Provisioning' ),
```

Comment:
Isn't `Provisioning` too technical for a user? I'd expect this should be something more user-friendly, like "Configuring" or "Setting up"

Reply:
In this case I am preserving the copy from the original BackupScan component so I think it is okay to leave it. However I do think that when we release Jetpack Scan, we should rethink all of this including the copy.